apps/condo/domains/billing/schema/BillingIntegration.test.js

```diff
@@ -37,6 +42,49 @@ describe('BillingIntegration', () => {
             expect(updatedIntegration.dataFormat.hasToPayDetails).toEqual(true)
         })
 
+        test('update group with wrong payload', async () => {
+            const admin = await makeLoggedInAdminClient()
+            const [objCreated] = await createTestBillingIntegration(admin)
+
+            const errMsg = `${WRONG_FORMAT}:BillingIntegration:group] group should be a sequence of lowercase latin characters`
```

Comment:
not reliable message...Since it typed in 2 places...
How about moving it to consts?

Reply:
Nice idea, moved this to consts